### DISCUSSION TITLE
Rename Classic scoring to avoid confusion with Classic mod

### DISCUSSION THREAD

**Discussion by WitherFlower** [ppy/osu#13197] [Ideas]
To give a bit of context : I was having a discussion about scoring in lazer in a discord, and I noticed people easily confused Classic scoring and Classic mod, because they have very similar names.

**Describe the new feature:** 

Rename Classic scoring to avoid confusion with Classic mod

**Proposal designs of the feature:** 

Renaming `Classic` scoring to `Legacy` scoring seems like a fine solution, this way we have two different terms for two different things.

**Comment by bdach** [comment]
I'm not super sure that having "legacy scoring" and the "classic mod" alongside each other is much better, either. They'd be two terms for essentially the same thing which is compatibility with stable, but in different areas (scoring vs gameplay mechanics). I'm not saying the current state of affairs is great but the proposed change does not feel noticeably better.

I could perhaps get behind having classic mod also prefer/force classic scoring somehow, but that's my personal opinion. Not sure what should happen to scorev2 plays from stable if that was to be done, though.

**Comment by LittleEndu** [comment]
I would like to suggest using a synonym of "classic" for scoring. Something like "Traditional" might work, this would also make sense as traditionally other rhythm games would use this kind of scoring.

Renaming scoring to "Legacy" feels like kicking the can down the road. "Classic mod" and "Legacy scoring" might not be confused with each other, but now "Legacy scoring" and scoring used by stable might.

**Comment by ribbanya** [comment]
What about just "v1"? I'm pretty sure the community is already familiar with "v2" as the name of the initial pass of Lazer's scoring system. Even if Lazer has moved beyond v2 by now, "v1" still clearly refers to Stable's system.

**Comment by WitherFlower** [comment]
Classic scoring in lazer is different from scorev1, peppy said in the past that scorev1 was to be discontinued

**Comment by ribbanya** [comment]
Ah, I see. My initial response would be to call it "v1.5" but that's getting confusing already.

**Comment by peppy** [comment]
I'm not sure this needs to change as it *does* aim to emulate classic scoring, just like classic mod aims to emulate classic gameplay.

If we are going to rename, I'd propose it is not named a synonym of classic (why??) but something like "combo heavy"? But I'm not sure I agree with the rename requirement in the first place.

**Comment by sr229** [comment]
Honestly, I don't see the confusion anywhere, in a UX perspective, this doesn't need changing, most likely you just got confused on the intent of the Classic scoring vs Classic mod. They're two different entities and shouldn't be related too much.

**Comment by LittleEndu** [comment]
> (why??)

I also think classic mod and scoring are both trying to achieve similar goals and wanted to keep that similarity apparent. But I also agree that it shouldn't be that confusing to even warrant renaming.
